week-3/slideshow/index.html

```diff
@@ -3,12 +3,25 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Slideshow</title>
```

Comment:
Bugs & issues encountered. 
The site breaks at certain instances. Steps to reproduce the bug - 

1. Launch index.html.
2. Press the Auto Forward button.
3. Press the Auto Backward Button.
4. The website glitches and gets stuck between two given images.

What is the expected behaviour in this case? And how would you fix this issue?

Reply:
To further the above experiment. If I press the auto forward / backward button and soon after I press any other buttons. What would be the expected behaviour in that case? Does your site exhibit the expected behaviour?